src/main/java/com/global/api/builders/AuthorizationBuilder.java

```diff
@@ -598,6 +602,11 @@ public AuthorizationBuilder withUniqueDeviceId(String value) {
         uniqueDeviceId = value;
         return this;
     }
+    
+    public AuthorizationBuilder withEstimatedTransactions(Integer estNumTxn) {
+        this.estimatedNumTxn = estNumTxn;
+        return this;
+    }
```

Comment:
If we update the management builder to use a single method, we'll probably want change this to match.